wiki/Shortcut_Key_Reference/en.md

```diff
@@ -1,179 +1,199 @@
-
-![osu! List of possible mods (Key binding in order)](Soloplay8.png "osu! List of possible mods <Key binding in order>")
-
--   **Q**: [Easy](/wiki/Game_Modifiers)
--   **W**: [No Fail](/wiki/Game_Modifiers)
--   **E**: [Half-Time](/wiki/Game_Modifiers)
-
-------------------------------------------------------------------------
-
--   **A**: [Hard Rock](/wiki/Game_Modifiers)
--   **S**: [Sudden Death](/wiki/Game_Modifiers)
--   **D**: [Double Time](/wiki/Game_Modifiers)
--   **F**: [Hidden](/wiki/Game_Modifiers)
--   **G**: [Flashlight](/wiki/Game_Modifiers)
-
-------------------------------------------------------------------------
-
--   **X**: [Relax](/wiki/Game_Modifiers) (No effect on osu!mania's 4/5/6/7/8K mod)
--   **C**: [Auto-Pilot](/wiki/Game_Modifiers)
--   **V**: [Spun-Out](/wiki/Game_Modifiers)
--   **B**: Auto
-
-Play Mode
----------
-
--   **+/-**: Adjust local offset (hold Alt to change the precision)
--   **Escape**: Pause (Quit song if Auto mod is used)
--   **Spacebar**: Skip opening cut-scene.
--   **Tab**: Toggle scoreboard
--   **Shift+Tab**: Toggle in-game interface
--   **F3/F4**: Adjust speed for osu!mania.
--   **Ctrl+R or \`**: Quick retry the current map (autoskips the cut-scene)
-
-Edit Mode
----------
-
-### Keyboard Shortcuts
-
-#### General
-
-![How it would look like when you press H (Help) key](Edit_Help.jpg "How it would look like when you press H <Help> key")
-
--   **H**: Help (Contains all the shortcuts shown below)
-
-**Music**
-
--   **Z**: Stop
--   **X**: Play from start
--   **C or Spacebar**: Pause/Resume
--   **Left and Right arrow keys**: Skip through the song. Hold shift for 4x speed
-
-**Bookmarks commands**
-
--   **Ctrl+B**: Add bookmark at current location.
--   **Ctrl+Shift+B**: Remove bookmark at current location.
--   **Ctrl+Left Arrow**: Jump to previous bookmark.
--   **Ctrl+Right Arrow**: Jump to next bookmark.
-
-**Save and Reset**
-
--   **Ctrl+L**: Revert to saved
--   **Ctrl+S**: Quick save
--   **Ctrl+N**: Remove all notes (but keep timing/song setup)
-
-**Mode switch**
-
--   **F1**: Compose mode
--   **F2**: Design mode (only basics available at the moment)
--   **F3**: Timing mode
--   **F4**: Song setup
+# Shortcut Key Reference
+
+There are many shortcuts in osu! that can be utilized while ingame or while idle at any screen.
+They are listed out with their functions below.
+
+Note, some of these keys can be changed in the [Options](/wiki/Options/)!
+However, the list below uses the default keys.
+
+## Game-Wide
+
+These work anywhere:
+
+- `Shift` + `F1`: Open osu!status (when connection error)
+- `F7`: Toggle FPS limiting mode
+- `F8`: Toggle Chat Window
+- `F9`: Toggle Extended Chat Window
+- `F10`: Toggle all mouse buttons (execpt for mouse wheel)
+- `Ctrl` + `F11`: Toggle Frame Times
+- `F12`: Take Screenshot (hold `Shift` to upload instantly to server for sharing)
```

Comment:
L19: Revise, Suggested ``(Hold `Shift` to upload instantly to the server for sharing)``